### DISCUSSION TITLE
it is not remove installed application in windows 11 22000.739

### DISCUSSION THREAD

**Discussion by mumin16** [wixtoolset/issues#6814] [Questions] [answered]
It is not remove installed anything in windows 11 22000.739. There was no problem before
[Musavir-1.0.0-win64.zip](https://github.com/wixtoolset/issues/files/8987018/Musavir-1.0.0-win64.zip)
.

**Comment by wyrdfish** [comment, marked as answer]
Create a uninstall log and see what the error is.
